FlightPlanProject/FPSettings.cs

```diff
@@ -0,0 +1,58 @@
+using FlightPlan.Tools;
```

Comment:
This new class hold all settings that are not using Spacewarp settings. 
I've aligned on what is made on k2D2 to avoid the old style interface. 

each value in the Ui should point on one of those fields